vignettes/quality_control_workflow.Rmd

```diff
@@ -0,0 +1,306 @@
+These peptides are important to take into account because they can tell you if your tryptic digest was inefficient. 
+The number of missed cleavages should generally be low in a proteomics or LiP-MS dataset. 
+
+We are going to check the numbers of missed cleavages in our dataset by using the function `qc_missed_cleavages()`. The function can assess the missed cleavages based on the count of peptides with missed cleavages or the intensities of the corresponding peptides. For `method = "intensity"` the function uses the raw (not log2 transformed) intensity values. You can have the function either return a plot or a table. 
+
+```{r qc_missed_cleavages, fig.width = 6, fig.height = 4, fig.align = "center"}
+qc_missed_cleavages(data = data, 
+                    sample = sample, 
+                    grouping = peptide, 
+                    missed_cleavages = n_missed_cleavage, 
+                    method = "intensity",
+                    intensity = raw_intensity, 
+                    plot = TRUE)
+```
+
+### Sequence coverage
+
+The following function gives you insight into the protein coverage percentages in the dataset. For a LiP-MS experiment high sequence coverages are desirable. 
+
+To assess the distribution of the protein coverage, we are going to use the function `qc_protein_coverage()`. If you do not have a column containing the protein coverages in your data, you can use the function `sequence_coverage()` to obtain this information. 
+
+```{r qc_sequence_coverage, fig.width = 6, fig.height = 4, fig.align = "center"}
+qc_sequence_coverage(data = data, 
+                     protein_identifier = protein, 
+                     coverage = coverage)
+``` 
+
+### Peak width
+
+In order to identify potential chromatographic issues that might have occured throughout the measurement process one can take a look at the peak widths over the complete chromatogram. To do this we are going to use the function `qc_peak_width()`. This function requires either the peak start and end times or the retention time and peak width. This information can be obtained from Spectronaut (or any other search engine of your choice). 
+The peak widths should be similar for all the measured samples of the experiment. 
+
+```{r qc_peak_width, fig.width = 6, fig.height = 4, fig.align = "center"}
+qc_peak_width(data = data,
+              sample = sample, 
+              intensity = peptide_intensity_missing,
+              retention_time = retention_time, 
+              peak_width = peak_width)
+``` 
+
+### Data completeness
+
+The following function `qc_data_completeness()` checks how many of all detected precursors, peptides or proteins were identified in each sample. The function uses raw intensities (not log2 transformed) and can return either a plot or a table. 
```

Comment:
```suggestion
The following function `qc_data_completeness()` checks how many of all detected precursors, peptides or proteins were identified in each sample. The function can return either a plot or a table. 
```
Actually it does not require raw intensities. should also be corrected in the function documentation. It does not matter if the intensities are transformed or not.